OutOfSchool/OutOfSchool.BusinessLogic/Services/Database/WorkshopService.cs

```diff
@@ -743,17 +746,20 @@ await ApplyFilterForMinistryAdmin(filter).
         return predicate;
     }
 
-    private async Task ApplyFilterForMinistryAdmin(WorkshopFilterAdministration filter)
+    private async Task<Expression<Func<Workshop, bool>>> PredicateForRoleMinistryAdmin()
     {
+        var predicate = PredicateBuilder.True<Workshop>();
```

Comment:
why do you need it here ?
you can create it in single line.
Isa it possible to pass all the data as a parameters ?

Reply:
Thank you.  I separated the logic of predicate building and database queries into different methods, so the data is passed as parameters.